doc/release-notes-1.1.0.rst

```diff
@@ -0,0 +1,621 @@
+.. _ncs_release_notes_110:
+
+|NCS| v1.1.0 Release Notes
+##########################
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+|NCS| v1.1.0 supports product development with the nRF9160 Cellular IoT device.
+It contains reference applications, sample source code, and libraries for Bluetooth Low Energy devices in the nRF52 Series, though product development on these devices is not currently supported with the |NCS|.
+
+.. important::
```

Comment:
Should we have this note?

Reply:
^^ good to have such a note here. nRF53 is not in the NCS master at the moment (but will come soon with the upmerge)